packages/utils/tokens/package.json

```diff
@@ -2,13 +2,16 @@
   "name": "@fluentui-react-native/tokens",
   "version": "0.23.1",
   "description": "Token interface parts and helpers",
-  "main": "src/index.ts",
-  "module": "src/index.ts",
-  "typings": "lib/index.d.ts",
-  "onPublish": {
```

Comment:
We had special logic in our scripts to handle this key, is it not there anymore?

Reply:
It was a beachball key to swap out package.json values during the publish process. It is a blind replacement and the code to do that still exists, but it is no longer leveraged at the moment. So we still have the capability.